src/components/CareerResourceList/OLDCareerResourceList.txt

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
What’s this text file?

Reply:
Oh that was just for my notes. I'll remove it

Reply:
Removed